WooCommerce/WooCommerceTests/ViewRelated/HubMenu/HubMenuViewModelTests.swift

```diff
@@ -4,10 +4,10 @@ import XCTest
 @testable import WooCommerce
 @testable import Yosemite
 
-@MainActor
```

Comment:
Move `@MainActor` to individual tests because putting this on the class causes a warning that is soon to be a failure in Swift 6.